emcc.py

```diff
@@ -1242,6 +1242,9 @@ def check(input_file):
       if shared.Settings.WASM:
         shared.Settings.BINARYEN = 1 # these are synonyms
 
+        if shared.Building.is_wasm_only():
```

Comment:
i think we should keep the asm.js file when debugging, as well (`EMCC_DEBUG=1`, etc.) - it's very useful there.